oauth2client/contrib/appengine.py

```diff
@@ -210,6 +211,35 @@ def create_scoped_required(self):
     def create_scoped(self, scopes):
         return AppAssertionCredentials(scopes, **self._kwargs)
 
+    def sign_blob(self, blob):
+        """Cryptographically sign a blob (of bytes).
```

Comment:
This doesn't differ at all from the overridden method doc string, right? It's a straight copy/paste?

How about `"""See AppAssertionCredentials.sign_blob for specification."""` instead?